syntax = "proto3";
import "Rectangle.proto";
import "Point.proto";
package streetgangpb;


message GetSceneResponse
{
	string track_id = 1;
	int32 result = 2;
	uint64 world_id = 3;
	Rectangle rect = 4;
	repeated Point items = 5;
}
